### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by grahamwhiteuk** [exaile/exaile#853] [Q&A] [answered]
Any idea when the next release is going to be made?  There's one or two changes I would like to bring into the Fedora package so I'm trying to gauge whether to wait for a release or backport a couple of patches to 4.1.2.

By the way, I tried asking this question on the Google Devel Group.  Are those groups still active?  If there's no response over there they might best be closed and removed from the web site.

Thanks,
Graham

**Comment by virtuald** [comment]
I haven't thought much about a next release, but I haven't been involved. @luzip665 ? 

Nobody uses the groups... but I thought they still worked. However, I haven't seen any emails from them. I think it would be best to move to github discussions however.

**Comment by luzip665** [comment]
Hi.

I would say we can build a new release.
But I would like to merge #842. #843, #844, #848, #849, #850, #851 before.

@virtuald What can we do if @mathbr is not answering to #849? Just merge?

**Reply by mathbr** [reply]
I'm still around and reading what's happening. ;-) In this case I went ahead and did the merge, thanks for the mention.

**Comment by luzip665** [comment]
Hi.

I'm sorry, something went wrong building the release.
I thought I added the tag with commit https://github.com/exaile/exaile/commit/b8e1753c1e14f5f3b5ba9f6b8558a1fed9a69ae2 but it was written to commit https://github.com/exaile/exaile/commit/96c57fb5b50484db062f538438ed1e95b3d01c47, which is the wrong one.

Additionally the fedora-test check failed.

**Comment by sjohannes** [comment]
I think you can simply retag the right commit and push that. Not sure if the CI will rerun it, or what happens to the GitHub Release - you may have to delete the release first and rerun CI. In the worst case, you can create an empty commit and tag that beta2.

> Additionally the fedora-test check failed.

It times out after 4 minutes downloading the Docker image. Probably just a network glitch; usually the whole CI runs in under 1 minute.

(Edit: Sorry, this should have been posted as a reply to @luzip665's post; I haven't used GitHub discussions much before.)

**Comment by luzip665** [comment]
Huh, this worked.
So, release is on the way.

**Reply by sjohannes** [reply]
Is the Windows installer correct? AppVeyor says the deployment failed, so the release might still be pointing to the old version.

**Reply by rokm** [reply]
Looks like we'll need to push beta2 anyway for Windows version to work (#859).

**Reply by luzip665** [reply]
> Is the Windows installer correct? AppVeyor says the deployment failed, so the release might still be pointing to the old version.

The problem was that AppVeyor wasn't able to upload the installer in the second run, because GitHub allready added a release because of my failed run before. 
So, I replaced it manually.

**Reply by luzip665** [reply]
> Looks like we'll need to push beta2 anyway for Windows version to work (#859).

Is this blocking that way?
I can build a new release

**Reply by rokm** [reply]
> I can build a new release

Sure, go ahead.

**Reply by luzip665** [reply]
Done

**Comment by luzip665** [comment]
I would like to release the next beta of 4.1.3.
Does something speak against it?

**Reply by genodeftest** [reply]
Sounds good to me.

PS Why not do a 4.1.3 release directly? The last release is almost 1 year old and there haven't been many changes since the last beta anyways.

**Reply by luzip665** [reply]
Releasing the 4.1.3 directly would also be possible.
The most important change is the handling of flac/ogg tags, but it's not necessary to use.

Other opinions?

**Reply by grahamwhiteuk** [reply, marked as answer]
I'd get a 4.1.3 full release out as well.  No need for lengthy beta cycles.

**Reply by luzip665** [reply]
Everything done.

BTW who is able to publish posts on https://groups.google.com/g/exaile-users? Last post there is from 2021

**Reply by grahamwhiteuk** [reply]
I'd suggest shutting down the groups, @virtuald mentioned above that they're not used any more.  Also get them removed from the web site and replaced with some information about how the developers can be contacted e.g. through the GitHub discussions feature.

**Comment by genodeftest** [comment]
There has been a release at 2023-08-03. I guess it is ok to close this discussion. There is another discussion #943 for another release. Feel free to reopen in case you think I'm wrong.
